HW1/P4/P4.py

```diff
@@ -0,0 +1,54 @@
+
+
+import pickle
+import string
+import re
+import urllib2
+import csv
+import itertools
+import math as ma
+import os
+import P4_bfs
+
+
+if __name__ == '__main__':
+
+
+	partition = 16
+
+	output_rdd_captain_america = P4_bfs.bfs(start_node = 'CAPTAIN AMERICA', number_interations = 10, partition = partition)

```

Comment:
It would probably be faster to generate the graph/adjacency list once, and pass it to the bfs function.